UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,25 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    while ( -not (Test-Path $LogFile -Type Leaf) ) {
```

Comment:
Good code, but can we move this test to the job? I'd prefer if this code only waited on the unity process to handle cases where Unity has errors with the logfile itself.

Reply:
inside the script block?

Reply:
Yeah, inside the script block.